### PR TITLE
add and fetch topics

### PR DESCRIPTION
**Pull request by Younggregs**

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "health-e-api"
+    "default": "health-e-1015d"
   }
```
Comment (SapeleD3) [inline .firebaserc:4]: why did you change the project name

File: functions/controllers/post/getTopic.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const _ = require('lodash');
+const HttpStatus = require('http-status-codes');
+const { message, status } = require('../../util/constants');
+
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus;
```
Comment (SapeleD3) [inline functions/controllers/post/getTopic.js:5]: const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = require('http-status-codes');
preferred approach now

File: functions/controllers/post/newTopic.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const { error, success } = status;
+const {
+    somethingWentWrong,
+    topicExists,
+} = message;
+const {
+    INTERNAL_SERVER_ERROR,
+    CREATED,
+    CONFLICT,
+} = HttpStatus;
```
Comment (SapeleD3) [inline functions/controllers/post/newTopic.js:16]: const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = require('http-status-codes');
preferred approach now

**Review by SapeleD3 — COMMENTED** [review]
awesome work  man .. implement the changes and merge this

File: functions/controllers/post/newTopic.js (hunk 3 above)
Comment (SapeleD3) [reply functions/controllers/post/newTopic.js:16]: also fixing the httpstatus code import would resolve the conflict

**Review by EngrEric — CHANGES_REQUESTED** [review]
why did you change the project name in firebase
